UnityProject/Assets/Scripts/Objects/Disposals/DisposalMachine.cs

```diff
@@ -207,7 +207,7 @@ protected void TryUseWrench()
 
 		private void UseWrench()
 		{
-			objectPhysics.SetIsNotPushable(objectPhysics.IsNotPushable);
+			objectPhysics.SetIsNotPushable(!objectPhysics.IsNotPushable);
```

Comment:
this double negation will bite us in the ass eventually

Reply:
What do you mean double? its only toggling once

Reply:
read it in English, ``not is not pushable``. Issue isn't your code directly but the bool property

Reply:
should I rename the property? will most likely have to change a few scripts but should be fine

Reply:
would have to see all usages and find if it is fine to invert

Reply:
Visual studio says 31 references across 21 scripts

Reply:
also all inspector vlaues on prefabs would have to be inverted

Reply:
It's really about whether an object can move.

I suggest `IsAnchored()` which wouldn't require updating prefabs or if that sounds too machiney, `IsMovable()`.